DEPENDENCIES.rst

```diff
@@ -106,6 +114,14 @@ and (optionally):
 #. libblas-dev
 #. liblapack-dev
 #. libgoogle-perftools-dev
+#. python3
+#. python3-dev
+#. python3-tables
+#. python3-pandas
+#. python3-numpy
+#. python3-nose
+#. python3-jinja2
+#. cython3
 
```

Comment:
I'd rather not. This is a fully valid install and apt should not mess up anyone's system with this.  Also, I am pretty sure that on newer versions of ubuntu python3 is the default, not python2